src/tests/tasks.test.js

```diff
@@ -1,18 +1,96 @@
 import request from 'supertest';
-import Project from '../models/Projects';
-import projectsSeeds from '../seeds/projects';
 import app from '../app';
+import Tasks from '../models/Tasks';
+import task from '../seeds/tasks';
 
 beforeAll(async () => {
-  await Project.collection.insertMany(projectsSeeds);
+  await Tasks.collection.insertMany(task);
 });
 
-describe('Test Projects routes', () => {
-  test('It should get the project list', async () => {
-    const response = await request(app).get('/projects');
-    expect(response.body.message).toBe('Project found');
+describe('Test Tasks routes', () => {
```

Comment:
En los describe por lo general va la ruta en especifico que se va a testear, por ej
```suggestion
describe('GET /tasks/get', () => {
```

Reply:
Claro deberias tener un describe por cada verbo